### DISCUSSION TITLE
Prospective Community Forum Options

### DISCUSSION THREAD

**Discussion by techiegirl2day** [OpenSageTV/Community#15] [General]
https://itsfoss.com/open-source-forum-software/

Thought I would offer some suggestions. I am sorry I didn't get to post this in old forum before it was archived.   Maybe something in the link would be a good fit?

Galaxysurfer

**Comment by jusjoken** [comment]
I think that ship has sailed but any of those solutions require a cloud based server paid for by the remaining forum members which can not only add up in cost but can be difficult to coordinate payment.  Hopefully these GitHub Discussions will suffice.

---
@jusjoken
If you wish to see what I am up to and support my efforts visit my [Patreon page](https://www.patreon.com/user?u=58035567)

**Reply by techiegirl2day** [reply]
OK was trying to continue conversation here since the other forum goes to read only mode on June 1. 
As to visiting your Patreon, Sorry am not able to contribute to anything financially at this time. That may change in the future.
